src/main/kotlin/bingsoochef/bingsoochef/toppping/presentation/ToppingController.kt

```diff
@@ -11,30 +11,30 @@ import org.springframework.web.bind.annotation.*
 
 @RestController()
 @RequestMapping("/users/bingsoos/toppings")
```

Comment:
앞에 버전같은 정보를 주가해주는건 어떨까요?
또 굳이 users가 필요한 이유가 있을까요?

```suggestion
@RequestMapping("/api/v1/bingsoos/toppings")
```

Reply:
API Versioning은 프론트엔드 개발 시작 후에 다시 논의해봅시다!
URI는 빙수라는 자원을 사용자가 가지고 있는 관계라 이렇게 지었습니다.